issuer-list/src/main/java/com/adyen/checkout/issuerlist/IssuersUtils.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by atef on 25/8/2022.
+ */
+
+package com.adyen.checkout.issuerlist
+
+import com.adyen.checkout.components.model.paymentmethods.InputDetail
+import com.adyen.checkout.components.model.paymentmethods.Issuer
+
+fun getIssuers(issuerList: List<Issuer>): List<IssuerModel> =
```

Comment:
My issue with this extension method is that you can call `getIssuers` from anywhere in the project (this module or anything that extends it) without much context, you start writing `get` and you see this method that is 99% not relevant to you.

I suggest we make these methods `internal` to begin with and change the name to maybe `mapToModelList` to better explain what it does.
And also change them from `fun getIssuers(issuerList: List<Issuer>): List<IssuerModel>` to `fun List<Issuer>.mapToModel(): List<IssuerModel>` so that you can only call the method on a list of issuers and not just anywhere.

WDYT?

Reply:
good point actually